src/charts.js

```diff
@@ -39,15 +39,15 @@
     var LINE_STROKE_WIDTH = '1px';
     var POINT_STROKE_WIDTH = '1px';
 
-    if( !$element.jquery ){
-      $element = $( $element );
+    if( !$element.node ){
```

Comment:
Potential breaking change here. Since the Chart() function was originally designed to accept jQuery objects as well as DOM nodes, we need to first check if jQuery is loaded and then check if the element passed in is a jQuery object before calling d3.select on it. Since the d3.select() function will not accept a jQuery object, any jQuery object passed in must first be converted to a DOM node.